src/Display/Tooltip/Tooltip.tsx

```diff
@@ -15,21 +15,40 @@ const Tooltip: React.FunctionComponent<Props> = ({
   position,
   ...defaultProps
 }) => {
+  const ref = React.useRef(null);
   const [isHover, setIsHover] = React.useState(false);
   const showTooltip = () => setIsHover(true);
   const hideTooltip = () => setIsHover(false);
 
+  const touchOutside = (event: TouchEvent) => {
+    const element = event.target as HTMLElement;
+    const hasTouchedOutside = !ref.current.contains(element);
+
+    if (hasTouchedOutside) {
+      hideTooltip();
+      document.removeEventListener('touchstart', touchOutside);
+    }
+  };
+  const handleTouchStart = (event: React.TouchEvent) => {
+    const eventType = event.type as string;
+
+    showTooltip();
+
+    if (eventType === 'touchstart') {
```

Comment:
why need to check eventType === 'touchstart'?

Reply:
I forgot to remove. Thanks!